### PR TITLE
Update RELEASE.md

### PR DESCRIPTION
**Pull request by roberth**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
+- git commit -m "hercules-ci-agent-X.X.X"
+- wait for CI to succeed
+- git log -2
+- copy the contents of the api changelog for this release to: git tag -s -a hercules-ci-api-Y.Y.Y.Y API_COMMIT
+- copy the contents of the agent changelog for this release to: git tag -s -a hercules-ci-agent-X.X.X AGENT_COMMIT
 - git push --tags
-- bump agent version in for-upstream/common.nix package option
 - open a PR against master
 - open a PR against stable using the tag as a base
+- create/update nix-darwin PR
```
Comment (domenkozar) [inline RELEASE.md]:
```suggestion
- bump version in nix-darwin
```

Comment (roberth) [reply RELEASE.md]: What about the common files? Those should be updated.

Comment (domenkozar) [reply RELEASE.md]: Right, so it's sync common files with nix-darwin and bump the version.

Comment (domenkozar) [reply RELEASE.md]: Also needs to test if default package works (we override it)

**Comment by domenkozar** [comment]
- we forgot to add 0.3.0 url in changelog so that should be part of the process as well

File: RELEASE.md [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
-- using git log update hercules-ci-agent/CHANGELOG.md
-- git commit -m "hercules-ci-agent-0.X.X" # or PVP for api package
-- copy the contents of the changelog for this release to: git tag -s -a hercules-ci-agent-0.X.X
+- git checkout -B vX.X.X (new agent version)
+- update hercules-ci-api/CHANGELOG.md
+   - git log PREVIOUS..HEAD -- ./hercules-ci-api
+- bump api version
+   - CHANGELOG.md
+   - hercules-ci-api/hercules-ci-api.cabal
+   - hercules-ci-agent/hercules-ci-agent.cabal
```
Comment (domenkozar) [inline RELEASE.md]:
```suggestion
   - hercules-ci-agent/hercules-ci-agent.cabal
- make sure CHANGELOG.md has a markdown link for the added version tag
```

**Comment by domenkozar** [comment]
Added more suggestions.

**Pull request merged**